test/tools/IsolTestOptions.h

```diff
@@ -30,11 +30,13 @@ namespace test
 
 struct IsolTestOptions: CommonOptions
 {
-	bool noColor = false;
 	bool showHelp = false;
+	bool noColor = false;
+	std::string runTest = std::string{};
 
 	IsolTestOptions(std::string* _editor);
 	bool parse(int _argc, char const* const* _argv) override;
+	void validate() const override;
```

Comment:
```suggestion
	void validateTest() const override;
```
Or smth like that?

Reply:
This overrides `CommonOptions::validate()` and would take care of all options, not only `--test` like in this case.

Reply:
Ok!